.github/workflows/tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Install CocoaPod dependencies
         run: pod install
       - name: Run UI Tests
-        run: set -o pipefail && xcodebuild -workspace 'Braintree.xcworkspace' -sdk 'iphonesimulator' -configuration 'Release' -scheme 'UITests' -destination 'name=iPhone 16,OS=18.2,platform=iOS Simulator'  test | ./Pods/xcbeautify/xcbeautify
+        run: set -o pipefail && xcodebuild -workspace 'Braintree.xcworkspace' -sdk 'iphonesimulator' -configuration 'Release' -scheme 'UITests' -destination 'name=iPhone 15,OS=17.5,platform=iOS Simulator'  test | ./Pods/xcbeautify/xcbeautify
```

Comment:
It seems like there are some bugs with GitHub actions version of iOS 18. I _am_ running this version locally and not running into these failures but it was having issues with our 3DS UI tests as well as PayPal UI tests. Again no actual issues on iOS 18 devices or simulators so just seems to be a runner issue. We can keep an eye on it and bump this back up down the line if they become more stable.